frontend/packages/dev-console/integration-tests/cypress.json

```diff
@@ -19,7 +19,7 @@
   "screenshotsFolder": "../../../gui_test_screenshots/cypress/screenshots",
   "videosFolder": "../../../gui_test_screenshots/cypress/videos",
   "env": {
-    "TAGS": "@smoke and not (@manual or @to-do or @un-verified)",
+    "TAGS": "(@pre-condition or @smoke) and not (@manual or @to-do or @un-verified or @bug)",
```

Comment:
Don't we need regression tag?

Reply:
Regression takes more time. First lets integrate this and later on we can add regression test scenarios as well based on the execution time